pyquidax/quidax.py

```diff
@@ -60,5 +60,30 @@ def withdrawal_fee(self, currency: Currency):
         )
 
 
-class AsyncQuidax:
-    ...
+class AsyncQuidax(BaseAsyncAPIWrapper):
```

Comment:
You didn't checkout from your main branch to work on this issue. This changes exist in your old pr an that's bad